app/controllers/users/reset_passwords_controller.rb

```diff
@@ -90,7 +90,12 @@ def request_id
     end
 
     def handle_valid_email
-      create_account_if_email_not_found
+      RequestPasswordReset.new(
+        email: email,
+        request_id: request_id,
+        analytics: analytics,
+        irs_attempts_api_tracker: irs_attempts_api_tracker,
+      ).perform
 
       session[:email] = email
```

Comment:
is this still needed?

Reply:
I think so. Or at least we currently validate on it to prevent accessing the confirmation "Check your email" page.

https://github.com/18F/identity-idp/blob/60a4c2ce2817526239c266b70c59bca1aefa55c6/app/controllers/forgot_password_controller.rb#L5-L6

![image](https://github.com/18F/identity-idp/assets/1779930/6476291e-5bf7-4c82-9c78-6ea705042666)

Reply:
Never mind, I see that it is

Reply:
Oops, yep, I didn't refresh to see your comment before I commented, but I found the same thing.